### ISSUE TITLE
psutil not included as dependency in version >= 0.2

### ISSUE THREAD

**Issue by amcraig** [Kanaries/pygwalker#188]
Hi all, 
I pulled the most recent Pygwalker 0.2.1, but it looks like not all the dependencies were spoken for on the jump from 0.1 to 0.2 and psutil needs to be included. 

```
pygwalker  | Traceback (most recent call last):
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
pygwalker  |     exec(code, module.__dict__)
pygwalker  |   File "/srv/app.py", line 6, in <module>
pygwalker  |     import pygwalker as pyg
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/pygwalker/__init__.py", line 14, in <module>
pygwalker  |     from pygwalker.api.walker import walk
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/pygwalker/api/walker.py", line 11, in <module>
pygwalker  |     from pygwalker.utils.execute_env_check import check_convert
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/pygwalker/utils/execute_env_check.py", line 1, in <module>
pygwalker  |     import psutil
pygwalker  | ModuleNotFoundError: No module named 'psutil'
```

Additionally if I resolve the psutil package myself I run into another issue with the execute_env_check function on my ubuntu linux VM:

```
pygwalker  | Traceback (most recent call last):
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 552, in _run_script
pygwalker  |     exec(code, module.__dict__)
pygwalker  |   File "/srv/app.py", line 46, in <module>
pygwalker  |     pyg_html = pyg.walk(df, spec=VIS_SPEC, env='Streamlit', hideDataSourceConfig=False)
pygwalker  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/pygwalker/api/walker.py", line 79, in walk
pygwalker  |     if check_convert():
pygwalker  |        ^^^^^^^^^^^^^^^
pygwalker  |   File "/usr/local/lib/python3.11/site-packages/pygwalker/utils/execute_env_check.py", line 9, in check_convert
pygwalker  |     cmd_list = psutil.Process().parent().cmdline()
pygwalker  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pygwalker  | AttributeError: 'NoneType' object has no attribute 'cmdline'
```

**Comment by longxiaofei** [comment]
Thanks for feedback.

You can use `pyg_html = pyg.walk(df, spec=VIS_SPEC, return_html=True, hideDataSourceConfig=False)` to get html of pygwalker.

About bug of this issue, I will fix it in next version.

**Comment by longxiaofei** [comment]
fixed in version 0.2.2.
